pkg/lightning/config/config.go

```diff
@@ -158,7 +161,7 @@ func (d *DBStore) adjust(
 	s *Security,
 	tlsObj *common.TLS,
 ) error {
-	if i.Backend == BackendLocal {
+	if i.Backend == BackendLocal || i.Backend == BackendRemote {
```

Comment:
Can this conditional statement be encapsulated into isPhysicalBackend

Reply:
please check this comment @zeminzhou